doc/manual/install-domserver.rst

```diff
@@ -179,6 +179,22 @@ this approach is that the webserver is not aware of the actual client IP. This
 means that access logs for the webserver will still report the IP of the proxy
 or loadbalancer.
 
+Scaling the domserver
+---------------------
+
+Documenting all possible ways to properly size the domserver is out of scope for this
+manual. But properly think about the memory management for the PHP-FPM children, we advice
+40 FPM children per gigabyte of RAM memory so around 500 for a 16Gb system.
```

Comment:
I wonder if these numbers are still accurate? I think when I came up with this number I just ran htop on a running server (maybe make it busy by forcing some judgehosts to do some rejudging) and noted the rss for the fpm processes, and rounded to some nice number. But maybe it'd be good to double check that.

Reply:
I think I can do that after the finals, sounds like a reasonable way for testing.

Reply:
Though I think 40/GiB is fine (we use a similar metric at $work, though the workload there is quite different) it might be better to change this to number of requests per second. The section title (which is great!) is regarding scaling domserver, but the body of the text is about how to fully utilize your/a server by running domserver. The wiki-link to 'scaling and load testing' does a better job, though even that link is inconsistent with the statement here (taking 70MiB as a 'more modern' number).

To me it makes more sense to base the requirements of the domserver on the (maximum) number of concurrent users and/or requests, because the number of fpm worker is the limiting factor of how many requests domserver can handle concurrently. Deriving a baseline for this is also much easier since we can simply calculate the number of concurrent (php) requests from the access log. (timestamp + request duration)

Having said that. I think this is fine for now. Lets use the finals to gather some data.

nit: isn't it `we advise` instead of `we advice`?